### ISSUE TITLE
Multiple inFoalTo Parameters Return Empty on Search

### ISSUE THREAD

**Issue by dacarver** [Betzer09/HorseApp2#14]
When passing multiple inFoalTo horses to `SearchActiveListings`, zero listings are returned. An example:

A search with:

`"inFoalTo": "Boon Too Suen"` returns 1 listing

`"inFoalTo": "Boon San,Boon Too Suen,Boon N Crockett,Boons Sonscoot"` returns 0 listings

**Comment by clerenge** [comment]
Multiple inFoalTo searches was not something that was implemented but I added that in for ya. Last night I changed the listings that are in dev to the listings that are in production in preparation for publishing. So currently most of the listings except one in dev now do not have the inFoalTo set to anything. So I added one more with the inFoalTo property set as "clerenge" weird word I know but if you search with inFoalTo as: 

"Dee Fancy Invester, clerenge" it works!
